content/docs/guides/pipelines/build-component.md

```diff
@@ -172,3 +172,6 @@ See [an example](https://github.com/kubeflow/pipelines/blob/master/samples/xgboo
 You can also build lightweight components from Python functions. See the guide 
 to 
 [lightweight python components](/docs/guides/pipelines/lightweight-python-components).
+
+## Export metrics
+See [Pipeline metrics](/docs/guides/pipelines/pipelines-metrics)
```

Comment:
Let's use capital letters only for brand names. So, you could refer to "Kubeflow Pipelines", but when just referring to a pipeline like this, it's better to use lower case. Also, we need a full stop at the end of the line. Suggested code:

```
See the guide to [pipeline metrics](/docs/guides/pipelines/pipelines-metrics).
```

Reply:
done